change_notes/2024-01-17-fix-reported-fp-for-a2-3-1.md

```diff
@@ -0,0 +1,2 @@
+`A2-3-1`: ` cpp/autosar/invalid-character-in-string-literal`
+    - Exclude wide string literals and utf8 string literal.
```

Comment:
the smallest suggestion, do we want to add the references to issue numbers where the fp was reported? it seems we used to do this like [here](https://github.com/github/codeql-coding-standards/blob/main/change_notes/2021-07-15-base-class-slicing.md)

Reply:
That is a good suggestion. Let me add that.